tests/foreman/ui/test_location.py

```diff
@@ -37,8 +37,8 @@ def valid_org_loc_data():
          'loc_name': gen_string('utf8', 10)},
         {'org_name': gen_string('latin1', 20),
          'loc_name': gen_string('latin1', 10)},
-        {'org_name': gen_string('html', 20),
-         'loc_name': gen_string('html', 10)}
+        {'org_name': gen_string('html', 1),
```

Comment:
I think that 2 should be fine, but more needs to be proved by huge amount of re-tests